presto-sheets/src/test/resources/gsheets-credentials.json

```diff
@@ -0,0 +1,12 @@
+{
+  "type": "service_account",
+  "project_id": "prestotest",
+  "private_key_id": "2101b30d636ee7ecdfac7e73bb98f95c669919fd",
+  "private_key": "-----BEGIN PRIVATE KEY-----\nMIIEuwIBADANBgkqhkiG9w0BAQEFAASCBKUwggShAgEAAoIBAQCtctiwbkigK47o\n4wTaISIMpAZMuzWHxzaxjwQyF3fOwAFRZY4Vo299dZ/sx7/uOx6ND2dHl1YkbYtf\niQXCqO1Pe2dzIkF0wdxv9q4lPN7FGnv64f/qipHIQhtIle/rtNFbF6T8YUzOO/gE\nh4LpbbsttsXoQQD8znutcyuX17JiWDqZAI23ek9oPkffnGH9UUH43fhwEandVHov\n0vtJreGwNLiyidKlGUFISCOVT/JbZG2h7GlVotIEzgD/Et/lDP5v+3R1gBfBEXkd\nLGZP9WjhDQdqiUbvfbsoaQqH4bEkeC1qCGax1qVi7/Ezj63B8Dj7DXWsL5eO4ZPc\nG9YKvh2ZAgMBAAECgf8aa6bVwN7FX8u074YfvL5cD/EnXJOxt4TZd9z2JKxTG1d8\ns9U3965psuS3t6QqBzEVOigXMFenulj+bERstIedU0xyFM1m/Lnt0OQEzqjNRVJ6\nXCCcOlFwuIblkXswAjti2D7Y0G+f61BIn8tUfV9GVtPDxPDHT/TaFZW21vvvfb2Q\nB7ahToHkDoaGMrEwoyyY3ZefAs26w4iMcmzYr/NwkLi9nFLSZqTX/i0f7/8x7SQb\nJnuALukGserl6eQnm8+8s85m6v6971SZMijGentWsVJ95cCwJkevhHxVqU80s+tt\nRMj4YgKHQOe9bMLhfMBXr+8nWBFW2FCUjLfT4AkCgYEA17BHX7WtGw6w8GealQgY\nqHxRXfXM8nre3vKm+U2JDEA4ykOi3v/HCyrJkxgGaKMGBEfdRQVjEgZI3BflH8ia\nzspsIg/ehqHqcla51az0UqzpnVA9OBUCquk+p3MoYqt9qkNfE1HVuwaiKFAr0jQm\n1IQz+R922fL96vUbQlS+HfUCgYEAzd2TmQzdG1GcybzRKf5OwbFBYSojvak/ciK7\nn7f2MLUd20FDj19A5tNC2ERRk63ugtZdbEreerLsQBDwLbF5VmvJz9iWLcShZzSO\nwGgOA/ZlC2nK+Nugq311LcP/rSjNTkIgCZC6Cg5vEq/nHkDiR3qB0nzeK8iQgRs7\nf06RNpUCgYBLWKs2RhL4tsMc2DRqB3AFu5f5CNrVghLFM2MdB/buLqkoD0SgPh3c\nStAr8GKcvwZlDG3BpMYdD9D4Aejql3ts7c2kgIKt6XYrBxzYBhk7oGfjqpoYM07B\nQcW/XT0oatv1VJ+r8hSFxyVyi78jKVC/QIefaVxterM1Fva0z0Gy2QKBgQC/LvrG\n1VJ173obRDY/uIPtj683DEp69bYNSkJSlyaUn28l0ttqR7zfYO0gKXpVGt95NJk4\nW+N3sbiD2vciiEI2Rw3575uwz35ReVblSv1RIr8+NTahnF3/RRjQssg8SLs55WEZ\nYQXVFwiCI0gRKZK7bQMptnsh0DabH+5GUXG8bQKBgHTORLZx9SpmSiotPWBTZA0K\nYShe6Ll13LAmX2ssfMJvLfnVHGvrzJZWvuiQFabrjaackIKRLOFQ+6+MuIn5DyMS\nr8KKZyoGxpLgRWmopDwdXdmtvqkUfmvpSRcaBvkhTLpy2qneaiFV+OIhE7DbOeRM\nUcR472gtJY0d4KTmaufK\n-----END PRIVATE KEY-----\n",
```

Comment:
Can this be distributed via confidant instead of committed into git?

Reply:
We need this for integration tests, not sure if we can load from confidant outside lyft. How about base64 encoding the credentials and keeping it under test resources?
(I have set up a google test account for this).

Reply:
For production let's be sure we use confidant. The values should be available on our aws hosts.